ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/SourceDescription.java

```diff
@@ -43,6 +43,7 @@
   private final String topic;
   private final int partitions;
   private final int replication;
+  private final String statement;
```

Comment:
needs including in `hashCode` as well.

This highlights the lack of testing of equals and hashCode in this class. Can you add a suitable unit test please?  Take a look at our use of `EqualsTester`, e.g. in `CreateStreamTest`, for an example of the testing pattern.